src/i18n/en-US/intake.ts

```diff
@@ -34,7 +34,7 @@ const intake = {
     BIZ_CASE_FINAL_SUBMITTED: 'Final business case received',
     READY_FOR_GRT: 'Ready for GRT meeting',
     READY_FOR_GRB: 'Ready for GRB meeting',
-    LCID_ISSUED: 'LCID: ',
+    LCID_ISSUED: 'Lifecycle ID issued: ',
```

Comment:
```suggestion
    LCID_ISSUED: 'Lifecycle ID issued',
```

The whitespace exists  in the code currently, but I don't like seeing this whitespace because it can be error prone. By looking at the keys around `LCID_ISSUED`, it looks like these are standalone statements. It might be better to add the `: ` where we interpolate the LCID.